cmd/gitopsUpdateDeployment.go

```diff
@@ -81,37 +82,46 @@ func gitopsUpdateDeployment(config gitopsUpdateDeploymentOptions, telemetryData
 }
 
 func runGitopsUpdateDeployment(config *gitopsUpdateDeploymentOptions, command gitopsUpdateDeploymentExecRunner, gitUtils iGitopsUpdateDeploymentGitUtils, fileUtils gitopsUpdateDeploymentFileUtils) error {
-	temporaryFolder, err := fileUtils.TempDir(".", "temp-")
+	err := checkRequiredFieldsForDeployTool(config)
 	if err != nil {
-		return errors.Wrap(err, "failed to create temporary directory")
+		return errors.Wrap(err, "not all required fields for this deploy tool are configured")
```

Comment:
I think you've already specified the error in detail and thus you could omit the error wrapping.